### PR TITLE
Widget Wizard second step implementation.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

**Comment by DzmitryHumianiuk** [comment]
@DiscoElevator up

File: app/src/components/main/modal/modalLayout/modalField/modalField.jsx [hunk 1]
```diff
@@ -4,25 +4,43 @@ import styles from './modalField.scss';
...
+  tip: PropTypes.string,
+};
+Tip.defaultProps = {
+  tip: '',
+};
+
 const Label = ({ label, labelWidth }) => (
-  <div className={cx('modal-field-label')} style={{ width: labelWidth || 'unset' }}>
+  <div
+    className={cx('modal-field-label', { 'no-min-height': label === ' ' })}
```
[the remark this answers is not in the document]
Comment (ArtsiomYeliseyenka) [reply app/src/components/main/modal/modalLayout/modalField/modalField.jsx]: This class required for cases when we have area for label, but it's empty. Field height, in that case, will be calculated by "modal-field-content" element height and can be less then 30px. So we need exact comparison with ` '   '`.

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/customColumnsControl/customColumnItem/customColumnItem.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  onChangePrefix = (e) => {
+    const { index, onChange, name } = this.props;
+    onChange({ name, value: e.target.value }, index - 1);
+  };
+  render() {
+    const { intl, name, prefix, index, last, noRemove } = this.props;
+    return (
+      <ModalField
+        className={cx({ 'last-custom-column-field': last })}
+        label={`${intl.formatMessage(messages.label)} ${index}`}
```
[the remark this answers is not in the document]
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/customColumnsControl/customColumnItem/customColumnItem.jsx]: fixed

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/differentLaunchesComparisonControls.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        STATS_TI_TOTAL,
+      ],
+      itemsCount: DEFAULT_ITEMS_COUNT,
+      metadata_fields: [METADATA_FIELDS.NAME, METADATA_FIELDS.NUMBER, METADATA_FIELDS.START_TIME],
+    });
+  }
+
+  render() {
+    return (
+      <Fragment>
```
[the remark this answers is not in the document]
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/differentLaunchesComparisonControls.jsx]: fixed

**Pull request merged**